pkg/operator/nfs/controller.go

```diff
@@ -117,22 +117,6 @@ func nfsOwnerRef(namespace, nfsServerID string) metav1.OwnerReference {
 	}
 }
 
-func getServerConfig(exports []nfsv1alpha1.ExportsSpec) map[string]map[string]string {
```

Comment:
I ended up removing this and replacing it with two lines in `createGaneshaConfig`, mainly because returning a map here will render the actual ordering of the exports written to the config to be indeterminable, since go maps are unordered. With the code changes I made earlier, this function wasn't doing much other than checking that `claimName` isn't empty and returning a map...  I think it makes more sense to check that no `claimName`s are empty with our up-front validation of the spec (though that's not added here, yet...)